src/server/isomorphic.ts

```diff
@@ -95,7 +95,11 @@ export default (context: IServerContext) => {
       }))
              .then(() => {
```

Comment:
can you format this a little bit different? Can you put `.all` one line down and align it?

```
      Promise
      .all(matchedComponents.map((component: Component) => {
...
      }))
      .then(() => {
...
      })
      .catch(...);
```

Reply:
I tried to not change the formatting at all but I'm happy to change it to the proposed version.